eng/versioning/version_client.txt

```diff
@@ -119,7 +119,7 @@ com.microsoft.azure:spring-data-cosmosdb;3.0.0-beta.1;3.0.0-beta.1
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 unreleased_com.azure:azure-messaging-servicebus;7.0.0-beta.5
-unreleased_com.azure:azure-security-keyvault-keys;4.2.0-beta.6
+unreleased_com.azure:azure-security-keyvault-keys;4.3.0-beta.1
```

Comment:
What uses this unreleased tag?

Reply:
Storage [here](https://github.com/Azure/azure-sdk-for-java/pull/14047/files#diff-9a69f999e24cab788eb291ec440a0622L112)